osfoffline/gui/qt/preferences.py

```diff
@@ -84,12 +89,22 @@ def set_containing_folder(self):
             return
 
         user = Session().query(User).one()
+
+        if save_setting:
+            # copy the synced folders from the old to new location
+            # so OSF doesn't think they were deleted
+            copy_tree(user.folder, os.path.join(osf_path))
+
         user.folder = os.path.join(osf_path)
 
         self.containingFolderTextEdit.setText(self._translate("Preferences", self.containing_folder))
```

Comment:
Since the textedit field doesn't actually have a text edit usage. Change https://github.com/CenterForOpenScience/osf-sync/blob/develop/osfoffline/gui/qt/generated/preferences.py#L38 to user Qlabel instead of QTextedit and change the naming of the related field to avoid confusion.